src/state-summary/state-card-input_text.html

```diff
@@ -26,6 +26,7 @@
         value='{{value}}'
         auto-validate='[[stateObj.attributes.pattern]]'
         pattern='[[stateObj.attributes.pattern]]'
+        type='[stateObj.attributes.mode]]'
```

Comment:
`mode` is not part of the backend config. Is there a PR to add it?

Reply:
Yes, there is. 
https://github.com/home-assistant/home-assistant/pull/11849